src/views/Nft/market/Collection/IndividualNFTPage/MainNFTCard.tsx

```diff
@@ -24,18 +24,49 @@ const Container = styled(Flex)`
 interface MainNFTCardProps {
   cheapestNft: NftToken
   cheapestNftFromOtherSellers?: NftToken
+  nothingForSaleBunny?: NftToken
 }
 
-const MainNFTCard: React.FC<MainNFTCardProps> = ({ cheapestNft, cheapestNftFromOtherSellers }) => {
+const MainNFTCard: React.FC<MainNFTCardProps> = ({ cheapestNft, cheapestNftFromOtherSellers, nothingForSaleBunny }) => {
   const { t } = useTranslation()
   const bnbBusdPrice = useBNBBusdPrice()
 
-  const nftToDisplay = cheapestNftFromOtherSellers ?? cheapestNft
+  let nftToDisplay = cheapestNftFromOtherSellers
+  if (!nftToDisplay) {
+    nftToDisplay = cheapestNft
+  }
+  if (!nftToDisplay) {
+    nftToDisplay = nothingForSaleBunny
+  }
+
```

Comment:
Can we rewrite this as `const nftToDisplay = cheapestNftFromOtherSellers || cheapestNft || nothingForSaleBunny`?